libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -607,6 +609,36 @@ private List<String> getFullNamesFromCredentials(List<IdentityClaim> identityCla
                 .toList();
     }
 
+    public boolean checkUncorrelated(String userId)
```

Comment:
Can we avoid the negative, and make it clear which way round it goes - perhaps `areVcsCorrelated` or similar?

At some point I'd like to avoid these methods re-fetching all the VCs every time, but that sounds like a separate ticket.

Reply:
I was thinking something similar but looks like a not insignificant amount of work

Reply:
I'll create a sub ticket unless told otherwise